src/main/java/org/jitsi/jicofo/xmpp/FocusComponent.java

```diff
@@ -466,7 +467,10 @@ else if (smackIq instanceof LogoutIq)
                         String.valueOf(authAuthority.isExternal())));
         }
 
-        if (focusManager.getJitsiMeetServices().getSipGateway() != null)
+        JigasiDetector detector
+            = focusManager.getJitsiMeetServices().getJigasiDetector();
+        if (focusManager.getJitsiMeetServices().getSipGateway() != null
```

Comment:
That property is advertised only once for the whole conference duration, so it should be true if there is possibility that Jigasi will be discovered ever. Anyway it may turn out some more work is needed for the availability handling on the clients side like it's done for Jibri. Or we can just send error if there are no Jigasis available at the time of request - eventually.